docker-entrypoint.sh

```diff
@@ -2,6 +2,13 @@
 set -e
 set -o allexport
 
+# Backup current env
+env > /tmp/environment.env
```

Comment:
The idea is that if you pass in environment variables directly with docker (`docker-compose run -e ENV_VAR=blah` they aren't overridden by the defaults, but at the same time you don't have to override anything to get it working out of the box.

There should be nothing we want to override in default.env but are unable to, as default.env is only for cabot specific environment variables

I tried to find a nicer way of doing this but couldn't find an alternative